### PR TITLE
feat: add canary checker client

### LINKED ISSUE #35: Client for canary-checker

### PR DESCRIPTION
**Pull request by simbatsuro**
add client for canary checker

Closes #35

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -20,29 +44,68 @@
...
-      IndicatorPanel
+      IndicatorPanel,
     },
 
     data () {
       return {
-        clusters: stats.clusters
+        apiclusters: null,
+        clusters: stats.clusters,
+        canary: {
```
Comment (moshloop) [inline src/App.vue:73]: This canary info/details should be coming from the API

File: src/App.vue [hunk 2]
```diff
@@ -20,29 +44,68 @@
...
+            {"id":20,"color":"#336600", "shape":"heart", "overlay":""}
+          ]
+        },
+
       }
+    },
+
+    mounted() {
+      this.$axios
+        .get('http://localhost:8080/api')
```
Comment (moshloop) [inline src/App.vue:105]:
```suggestion
        .get('/api')
```

File: src/App.vue (hunk 1 above)
Comment (simbatsuro) [reply src/App.vue:73]: Noted

File: pkg/get-canary.go [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+		Short: "Get canary information",
+		Args:  cobra.MinimumNArgs(0),
+		Run: func(cmd *cobra.Command, args []string) {
+			logger.Infof(`Retrieving Canary Info...
+				`)
+			Retrieva()			 
+		},
+	}
+
+func Retrieva() {
```
Comment (moshloop) [inline pkg/get-canary.go:23]: This method should be called by api.Serve method and replace the inline canary data

Comment (simbatsuro) [reply pkg/get-canary.go:23]: Noted

File: src/App.vue [hunk 4]
```diff
@@ -1,48 +1,60 @@
...
     </v-main>
   </v-app>
+
 </template>
 
+
 <script>
 
   import IndicatorPanel from './components/IndicatorPanel.vue'
-  import stats from './data/sample.json'
```
Comment (moshloop) [inline src/App.vue:34]: Can we keep this in, so that the demo preview page works ?

File: src/App.vue [hunk 5]
```diff
@@ -1,48 +1,60 @@
...
 
     data () {
       return {
-        clusters: stats.clusters
+        clusters: null,
       }
+    },
+
+    mounted() {
+      this.$axios
```
Comment (moshloop) [inline src/App.vue:53]: Support demo mode i.e. `if (DEMO_MODE) {this.clusters = demoData}`

Then in the index.html that gets deployed to netlify we set in the html
```html
<script>
var DEMO_MODE = true
</script>
```

And in the Docker image we  use a different index.html that doesn't have the DEMO_MODE enabled.

File: src/App.vue (hunk 4 above)
Comment (simbatsuro) [reply src/App.vue:34]: Yes we can

File: src/App.vue (hunk 5 above)
Comment (simbatsuro) [reply src/App.vue:53]: ok, thank you

**Pull request merged**